src/server/scripts/Kalimdor/zone_bloodmyst_isle.cpp

```diff
@@ -786,12 +786,17 @@ class spell_free_webbed : public SpellScript
 
     bool Validate(SpellInfo const* /*spellInfo*/) override
     {
-        return ValidateSpellInfo(SummonSpells);
+        return ValidateSpellInfo(
+        {
+            SPELL_FREE_WEBBED_1, SPELL_FREE_WEBBED_2, SPELL_FREE_WEBBED_3, SPELL_FREE_WEBBED_4,
```

Comment:
is this needed because ValidateSpellInfo doesn't support std::array ?

Reply:
Not supported or I don't know how to do it properly